No. Something is wrong somewhere.
![Screenshot (339)](https://github.com/CleverRaven/Cataclysm-DDA/assets/22739822/334ff13b-a6ce-4f37-af9d-d34ae6181fa8)
The land in the middle of the lake should be deep water (and that's the case when compiled with a master version with just a missing constant restored).
We'll see if I can figure out what's gone wrong here. I'm not too optimistic.
Encountered this while trying to figure out what this error report is supposed to mean
Error: /home/runner/work/Cataclysm-DDA/Cataclysm-DDA/src/map.cpp:8061:48: error: unnecessary temporary object created while calling emplace_back [modernize-use-emplace,-warnings-as-errors]
 8061 |                     loaded_grids.emplace_back( tripoint( gridx, gridy, gridz ) );
Sure, it compiles with  `loaded_grids.emplace_back( gridx, gridy, gridz  );` and seems to behave the same, but the compiler doesn't complain if I add another gridz, so I don't trust that.

--- a/src/map.cpp
+++ b/src/map.cpp
@@ -8058,7 +8058,7 @@ void map::shift( const point &sp )
                 loadn( point( gridx, gridy ), true );
 
                 for( int gridz = zmin; gridz <= zmax; gridz++ ) {
-                    loaded_grids.emplace_back( tripoint( gridx, gridy, gridz ) );
+                    loaded_grids.emplace_back( gridx, gridy, gridz );
                 }
             }
         }
@@ -8694,7 +8694,7 @@ void map::add_roofs( const tripoint &grid )
     for( int x = 0; x < SEEX; x++ ) {
         for( int y = 0; y < SEEY; y++ ) {
             const ter_id ter_here = sub_here->get_ter( { x, y } );
-            if( !ter_here->has_flag( "EMPTY_SPACE" ) ) {
+            if( ter_here.id() != ter_t_open_air ) {
                 continue;
             }
 

